### PR TITLE
Fix mediadb str issues

### PR DESCRIPTION
**Pull request by palfrey**
There's an issue with the use of `str` in mediadb, and a lack of tests for it meant it didn't turn up, so I've fixed that.

Needs https://github.com/opacam/Cohen3/pull/58 merging first for build fixes

### CODE REVIEW

**Comment by palfrey** [comment]
@opacam Any thoughts? I've just hit these again

**Review by opacam — APPROVED** [review]
**LGTM!!** :smile: , So many thanks and my apologies for not reviewing this before...

**Note:** I will leave this approved without merging for a few days, just in case that you want to address the comments. Otherwise don't worry,  I'll do it after the merging. **Thanks again!!!**

File: .github/workflows/push.yml
Comment (palfrey) [reply .github/workflows/push.yml]: Given the test break is due to tagpy not having a 3.6 wheel, and I'm the tagpy maintainer that would have to jump through hoops to fix this, entirely happy :) de10b811aa47165569f5e2ef041d094159156ba9 does this

File: tests/backends/test_mediadb_storage.py
Comment (palfrey) [reply tests/backends/test_mediadb_storage.py]: Done in e958c5a508fccbd8924de9440c58f7bbb86abf19

**Comment by opacam** [comment]
:rocket: Merged
Thanks again @palfrey !!!! :smile:

**Pull request merged**
